HotSpot/TableSearchViewController.h

```diff
@@ -0,0 +1,17 @@
+//
```

Comment:
you can delete TableSearchViewController.h and .m for now and create it again if you need it.